src/saml2/mdstore.py

```diff
@@ -930,9 +947,11 @@ def load(self, *args, **kwargs):
                 key = kwargs['url']
                 url = kwargs['url']
                 cert = kwargs.get('cert')
+                freshness_period = kwargs.get('freshness_period')
```

Comment:
I would suggest adding None as a second parameter, or deleting it from entity_transform to keep consistency.